runtime/flangrti/trace.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2006-2017, NVIDIA CORPORATION.  All rights reserved.
+ * Copyright (c) 2006-2019, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
No change?

Reply:
This appears to be an artifact of merging multiple Perforce changes into a single git commit. It looks like some of the Perforce changelists inserted changes and then backed them out, and the only change remaining was the copyright update.

I will try to keep these from showing up in Flang in the future.